docs/src/assets/scss/components/social-icons.scss

```
@@ -7,10 +7,10 @@
         padding: 0;
         margin-left: -1rem;
         margin-inline-start: -1rem;
+        display: flex;
```

```suggestion
        display: flex;
        justify-content: center;
```

Either we can `justify-content: center` or we can completely remove the style changes and leave it as it is before.